tests/scsi/008

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
+
+. tests/zbd/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test block data lifetime support"
+QUICK=1
+
+requires() {
+	_have_fio
+	_have_scsi_debug_group_number_stats
```

Comment:
_have_scsi_debug_group_number_stats() is missing.

I guess this helper function confirm that the group_number_stats sysfs file exists. It would be the better to make this helper generic for other sysfs attribute files. For example, I can think of the function as follows (untested):

    _have_scsi_debug_config() {
           local ret=0

           modprobe -q scsi_debug

           if ! path=$(_scsi_debug_key_path "${1}"); then
               SKIP_REASONS+=("scsi_debug does not have config ${1}")
               ret=1
           fi

           modprobe -qr scsi_debug
           return $((ret))
    }

Reply:
`_have_scsi_debug_group_number_stats()` has been added.